### DISCUSSION TITLE
How do I manage requests and enable data preloading?

### DISCUSSION THREAD

**Discussion by vaynevayne** [pmndrs/zustand#2725] [Ideas]
Similar to useSWR handling requests, but it relies heavily on keys and cannot be used outside components. Is there a kind person based on zustand encapsulated a hook to achieve a request for digital management? I feel it's mainly about implementing caching and timing state resets.

**Comment by steaks** [comment]
Hey @vaynevayne, I built a third party library that handles requests. Is this what you're looking for? https://github.com/steaks/leo-query
